ph5/clients/ph5tostationxml.py

```diff
@@ -140,19 +127,23 @@ def get_args():
 
     parser.add_argument("--latitude", action="store",
                         help=("Specify the central latitude point for a "
-                              "radial geographic constraint."))
+                              "radial geographic constraint."),
+                        type=float, dest="latitude", metavar="latitude")
```

Comment:
I think you don't need to specify ```dest```. For optional argument actions, the value of ```dest``` is usually inferred.